lib/internal/errors.js

```diff
@@ -79,9 +79,19 @@ module.exports = exports = {
 // Any error code added here should also be added to the documentation
 //
 // Note: Please try to keep these in alphabetical order
+E('ERR_ARG_NOT_ITERABLE', '%s must be iterable');
 E('ERR_ASSERTION', (msg) => msg);
 E('ERR_INVALID_ARG_TYPE', invalidArgType);
 E('ERR_INVALID_CALLBACK', 'callback must be a function');
+E('ERR_INVALID_FILE_URL_HOST', 'File URL host must %s');
+E('ERR_INVALID_FILE_URL_PATH', 'File URL path must %s');
+E('ERR_INVALID_THIS', 'Value of this must be of type %s');
+E('ERR_INVALID_TUPLE', '%s must be an iterable %s tuple');
+E('ERR_INVALID_URL',
+  (input) => `Invalid URL${input !== undefined ? `: ${input}` : ''}`);
```

Comment:
I think `undefined` should be in the message here..otherwise if the user makes a mistake and passes an undefined, there is no way to know if it is an undefined or just an empty string.